deliverable_3.md

```diff
@@ -0,0 +1,54 @@
+# System Description
+
+   The problem of lack of convenient and inclusive ways to game with others affects gamers. The Impact of which is a disconnected group of gamers. For gamers who need an online platform to connect with others, GameyTime is an online gaming platform that is an extremely convenient way to connect with other gamers: unlike other platforms that require you to download extra software or create an online account, our product will make it easier and faster for all gamers to game together. We aim to deliver a fast, easy to navigate and efficient browser-based gaming website that will offer a wide variety of games for free. The website is easily accessible for any type of person of any age and will allow multiple people to quickly connect and play together online. Our intended audience/consumer is casual gamers of any age who also have access to an internet connection and are looking to play a casual game with someone else online.
+
+   **Users** *connect* to the website by inputting the link www.gameytime.xyz (link tentative to change) to a browser. **Users** have the option to <ins>create</ins> a *profile* to the website which will <ins>store</ins> the user *rankings*, **games ratings**, and **achievements**. **Users** will <ins>select</ins> a single game **lobby** to join from a variety of possible **games**, which may include **tic-tac-toe**, **checkers**, and **mancala**. **Users** will enter the **game**, and <ins>play</ins> a **game** with another connected **user**. **Users** have the option to <ins>send</ins> a **message** through the use of an **in-game chat**, by <ins>selecting</ins> from a list of *pre-created messages* chosen by the website creators. **Users** are able to look at their *statistics* if they have registered a *profile* in addition to a **leaderboard** containing *rankings* from other **users**. User interface will be simple and clean.
+
+# Model
+![This is a UML case diagram png](/Images/gameytime-classdiagram.jpg)
```

Comment:
- The conceptual model does not have datatypes (e.g., String)
- review the use of arrows in the model
- the name of the classes are singular not plural
- some cardinalities are missing
- it is weird that the users do not have a name but in-game have a senderName. This means that theoretically, each message can have a different senderName even if they are from the same user
- Violating the DRY principle: gameTime, userName, gameTitle etc. are repeated in the model. Check all repetitions
- Misrepresenting inheritance - please, use the UML arrow for that
- I would expect to see an association between LeaderBoard and user
-

Reply:
We fixed our UML Diagram.
* Deleted datatypes
* reviewed arrows and fixed
* class name has been changed with singular
* missed cardinalities fixed
* deleted 'senderName'
* Followed DRY principle
* Fixed inheritance between classes
* added a association between LeaderBoard and User